pages/common/jp2a.md

```diff
@@ -0,0 +1,28 @@
+# jp2a
+
+> Convert JPEG images to ASCII.
+> More information: <https://csl.name/jp2a/>.
+
+- Read JPEG image from a file and print in ASCII:
+
+`jp2a {{path/to/image.jpeg}}`
+
+- Read JPEG image from a URL and print in ASCII:
+
+`jp2a {{www.example.com/image.jpeg}}`
+
+- Colorize the ASCII output:
+
+`jp2a --colors {{path/to/image.jpeg}}`
+
+- Specify characters to be used for the ASCII output:
+
+`jp2a --chars={{"..-ooxx@@"}} {{path/to/image.jpeg}}`
+
+- Write the ASCII output into a file:
+
+`jp2a --output={{file.txt}} {{path/to/image.jpeg}}`
```

Comment:
```suggestion
`jp2a --output={{path/to/file.txt}} {{path/to/image.jpeg}}`
```

Reply:
That maybe the right way but I was wondering if the output path and image file path may give rise to confusion for the readers.

Do see it that way.

Reply:
Otherwise, your suggestion is of course the right way to do it.

Reply:
idk, commands like this https://github.com/tldr-pages/tldr/blob/master/pages/linux/ksvgtopng5.md the output file must be in the current directory. So, I think is good say there is no limitation.

Reply:
Oh, good point. What about this?
```suggestion
`jp2a --output={{path/to/output_file.txt}} {{path/to/image.jpeg}}`
```
@sahilister @navarroaxel